### DISCUSSION TITLE
Идеи что бы pr3tty работал

### DISCUSSION THREAD

**Discussion by Kerman1337** [Rockstar234/MineareaUpdater#11] [Ideas]
* Сделать временную папку для скачивания файлов(архивов)
* Запуск игры сразу через батник, после установки
* Доработать метод через PrismLauncher
* Добавить кнопку для установки Java

**Comment by prettyleaf** [comment]
https://github.com/Rockstar234/MineareaUpdater/commit/16ceed099539bf3f9370ca8fe1b1913c51dabf1f
Launch button later

**Comment by prettyleaf** [comment]
> [16ceed0](https://github.com/Rockstar234/MineareaUpdater/commit/16ceed099539bf3f9370ca8fe1b1913c51dabf1f) Launch button later

Launch button was added in last [update ](https://github.com/Rockstar234/MineareaUpdater/releases/tag/1.0.5)
